classworks/AnnaVolodkova/01_git/result.txt 

```diff
@@ -0,0 +1 @@
+https://github.com/AnnaVolodkova/summer.git
```

Comment:
@AnnaVolodkova That's the wrong link. Please provide a link to the repo you made for solid-train exercise